doc/developer-guides/hld/virtio-net.rst

```diff
@@ -533,7 +533,7 @@ a PCI slot to the device model acrn-dm as shown below.
 
 .. code-block:: none
 
-   -s 4,virtio-net,<macvtap_name>,[mac=<XX:XX:XX:XX:XX:XX>]
+   -s 4,virtio-net,<macvtap=name>,[mac=<XX:XX:XX:XX:XX:XX>]
```

Comment:
@weichenli-intel Need to mention mac_seed?

Reply:
Yes, I have add some mention of mac_seed.